packages/uniforms-material/src/BoolField.tsx

```diff
@@ -1,3 +1,4 @@
+import { useTheme } from '@material-ui/core';
```

Comment:
Due to implied tree shaking, can we import `@material-ui/core/styles` or even `@material-ui/core/styles/useTheme` instead? Just like we do with the components.